requirements.txt

```diff
@@ -97,12 +97,6 @@ charset-normalizer==3.1.0 \
     --hash=sha256:f8303414c7b03f794347ad062c0516cee0e15f7a612abd0ce1e25caf6ceb47df \
     --hash=sha256:fca62a8301b605b954ad2e9c3666f9d97f63872aa4efcae5492baca2056b74ab
     # via requests
```

Comment:
```suggestion
    # via requests
# This requirement duplicates the line below, without an extra. We need this to
# work around https://github.com/pypa/pip/issues/9644 because we can't use
# --no-deps when this gets installed by GCF
google-api-core==2.17.1 \
    --hash=sha256:610c5b90092c360736baccf17bd3efbcb30dd380e7a6dc28a71059edb8bd0d8e \
    --hash=sha256:9df18a1f87ee0df0bc4eea2770ebc4228392d8cc4066655b320e2cfccb15db95
```